default.nix

```diff
@@ -44,6 +46,7 @@ let
             "--set JUPYTERLAB_DIR ${directory}"
             "--set JUPYTER_PATH ${extraJupyterPath pkgs}:${kernelsString kernels}"
             "--set PYTHONPATH ${extraJupyterPath pkgs}:${pythonPath}"
+            "--prefix PATH : ${extraPath}"
```

Comment:
Just a curiosity, this works with the space around the `:`? Isn't this parsed as three strings?

Reply:
Yes, this is the intended syntax for `makeWrapper`, as described in the [nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#fun-makeWrapper)

Reply:
So perfect, merging, thanks for the help!